fdbserver/FDBExecHelper.actor.cpp

```diff
@@ -169,17 +177,24 @@ ACTOR Future<int> execHelper(ExecCmdValueString* execArg, std::string folder, st
 		versionString += version;
 		paramList.push_back(versionString);
 		paramList.push_back(role);
-		cmdErr = spawnProcess(snapBin.toString(), paramList, 3.0, false /*isSync*/);
+		cmdErr = spawnProcess(snapBin.toString(), paramList, 3.0, false /*isSync*/, 0);
 		wait(success(cmdErr));
 		err = cmdErr.get();
 	} else {
 		// copy the files
 		state std::string folderFrom = folder + "/.";
 		state std::string folderTo = folder + "-snap-" + uidStr.toString();
+		double maxSimDelayTime = 1.0;
+		if (snapVersion == 1) {
+			folderTo = folder + "-snap-" + uidStr.toString();
+		} else {
+			folderTo = folder + "-snap-" + uidStr.toString() + "-" + role;
+			maxSimDelayTime = 10.0;
```

Comment:
snapshot create can get delayed to a max time of `maxSimDelayTime`. In v1, snapshots are assume to take only < 5 seconds, now it can take as much as ~10 seconds, what I really need is > 5 seconds, just to stretch it out - I have chosen a large number of 10 seconds.